dvc/repo/fetch.py

```diff
@@ -64,7 +64,7 @@ def _fetch(
             downloaded += out.get_files_number()
```

Comment:
Not sure which one of these raises CheckoutError, could you clarify?

Reply:
@Suor ?

Reply:
I've seen this when was fixing a test, I went other way in the end. I will try to reproduce it.